test/e2e/basic_test.go

```diff
@@ -116,18 +117,52 @@ func (data *TestData) testDeletePod(t *testing.T, podName string, nodeName strin
 	podIPs := podInterfaces[0].IPs
 	t.Logf("Host interface name for Pod is '%s'", ifName)
 
-	doesInterfaceExist := func() bool {
-		cmd := []string{"ip", "link", "show", ifName}
-		stdout, stderr, err := data.runCommandFromPod(antreaNamespace, antreaPodName, agentContainerName, cmd)
-		if err != nil {
-			if strings.Contains(stderr, "does not exist") {
+	var doesInterfaceExist func() bool
+	var doesIPAllocationExist func(podIP string) bool
+
+	if isWindowsNode {
+		doesInterfaceExist = func() bool {
+			_, err := hcsshim.GetHNSEndpointByName(ifName)
```

Comment:
I am confused, you cannot assume that this code is going to run on Windows. The test code itself can run on Linux, Windows or macOS.

Reply:
So you mean if we want to check Windows configurations in the test, need to use "RunCommandOnNode"？

Reply:
correct